esrally/mechanic/telemetry.py

```diff
@@ -790,40 +793,55 @@ def attach_to_node(self, node):
     def on_benchmark_start(self):
         if self.process is not None:
             self.process_start = sysstats.process_io_counters(self.process)
+            read_bytes = 0
+            write_bytes = 0
+            io.ensure_dir(self.log_root)
+            tmp_io_file = os.path.join(self.log_root, "%s.io" % self.node_name)
```

Comment:
Nit: As per convention we prefer the `format` method to `%` formatting in code that is not performance-critical, i.e. you'd need to change this to `"{}.io".format(self.node_name))` (similarly in `on_benchmark_stop`)